ui/pages/Search/selectors.js

```diff
@@ -154,6 +155,14 @@ export const getDatasetTypes = createSelector(
   },
 )
 
+export const getHasHgmdPermission = createSelector(
+  getUser,
+  getProjectsInput,
+  getProjectsByGuid,
+  (user, projectGuids, projectsByGuid) => user.isAnalyst || projectGuids.some(
+    projectGuid => (projectsByGuid[projectGuid] || {}).enableHgmd),
```

Comment:
Just some of the projects enabling HGMD access are enough to allow the user to access HGMD of all projects. Is it too loose in permission control?

Reply:
no, because if a PERSON has access to any project that we have agreed to grant HGMD access to, that means that PERSON has access to a HGMD license, which is what actually matters